vscode/src/ruby/rvm.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-process-env */
+import os from "os";
+import path from "path";
+
+import { asyncExec } from "../common";
+
+import { ActivationResult, VersionManager } from "./versionManager";
+
+// Ruby enVironment Manager. It manages Ruby application environments and enables switching between them.
```

Comment:
TIL it's not `Ruby Version Manager`!

Reply:
It used to be called that apparently https://rvm.io/rvm/myths.